code/modules/antagonists/wraith/critters/abilities/plaguerat.dm

```diff
@@ -1,3 +1,13 @@
+ABSTRACT_TYPE /datum/targetable/critter/plague_rat
```

Comment:
```suggestion
ABSTRACT_TYPE(/datum/targetable/critter/plague_rat)
```
I didn't even know this worked without parentheses. standard is to have them, regardless